mondrian/src/main/java/mondrian/rolap/RolapSchema.java

```diff
@@ -300,7 +301,11 @@ protected Logger getLogger() {
      * @param catalogUrl URL of catalog
      * @param catalogStr Text of catalog, or null
      */
```

Comment:
Were you going to leave the original load() method, and add the new one as an overload?

Reply:
Since the load method is protected and is not a public API, I was not concerned with maintaining the original load method.  I only overloaded methods that were public and we were changing the method signature.  I can overload the load method if you think it is necessary, but I personally do not see a reason to.

Reply:
It's possible that somebody has extended RolapSchema.  But I guess anybody who has customized to that extent could also adjust their implementation pretty easily to accommodate the new signature, so probably not a big deal either way.